models/staging/tpdm/base/base_tpdm__candidate_educator_preparation_program_associations.sql

```diff
@@ -0,0 +1,35 @@
+with candidate_educator_preparation_program_associations as (
+    {{ source_edfi3('candidate_educator_preparation_program_associations') }}
+),
+renamed as (
```

Comment:
Hi @zaharenka could you add new line between CTEs in all models please?

Reply:
as discussed, we'll leave this WITHOUT the blank line